lib/kube/proxy/forwarder_test.go

```diff
@@ -1106,7 +1104,10 @@ func TestKubeFwdHTTPProxyEnv(t *testing.T) {
 
 func newMockForwader(ctx context.Context, t *testing.T) *Forwarder {
 	clock := clockwork.NewFakeClock()
-	cachedTransport, err := ttlmap.New(defaults.ClientCacheSize, ttlmap.Clock(clock))
+	cachedTransport, err := utils.NewFnCache(utils.FnCacheConfig{
+		TTL:   5 * time.Hour,
```

Comment:
nit: this should be a named constant, or at least have an inline comment explaining what it is (this applies to a few other places as well).